spec/integration/discourse_slack/slack_spec.rb

```diff
@@ -219,6 +236,25 @@
             "filter" => "mute",
             "tags" => [tag.name]
           ])
+
+          post '/slack/command.json',
+            text: "status",
+            channel_name: "welcome",
+            token: token
+
+          json = JSON.parse(response.body)
+
+          expect(json["text"]).to eq(I18n.t(
+            "slack.message.status.all_categories",
+            channel: "#welcome",
+            command: DiscourseSlack::Slack.filter_to_present("mute")
+          ) + I18n.t(
+            "slack.message.status.with_tags",
+            tags: tag.name
+          ) + "\n" + I18n.t(
+            "slack.message.available_categories",
+            list: "uncategorized"
```

Comment:
"uncategorized" is a hard coded string here. The test will break if the uncategorized category is renamed.